### PR TITLE
use sync call with deadline in CrashManager instead of async

### PR DESCRIPTION
**Pull request by IrinaShkviro**
Fix for http://b/163018728

Replace asynchronous call with synchronous one (with deadline) for CrashManager

### CODE REVIEW

File: OrbitClientServices/CrashManager.cpp [hunk 1]
```diff
@@ -28,16 +30,19 @@ void CrashManagerImpl::CrashOrbitService(
...
 
-  std::function<void(grpc::Status)> callback = [](::grpc::Status status) {
-    if (!status.ok()) {
-      ERROR("Grpc call failed: %s", status.error_message());
-    }
-  };
-  crash_service_->experimental_async()->CrashOrbitService(&context, &request,
-                                                          &response, callback);
+  if (!status.ok()) {
+    ERROR("Grpc call failed: %s", status.error_message());
```
Comment (dimitry-) [inline OrbitClientServices/CrashManager.cpp:44]: we kind of expect it to fail with deadline aren't we? maybe check if it returns expected deadline exceeded error?

File: OrbitClientServices/CrashManager.cpp [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 namespace {
 
+constexpr uint64_t kTimeoutMilliseconds = 10;
```
Comment (dimitry-) [inline OrbitClientServices/CrashManager.cpp:12]: I would increase it a little bit - to 100ms probably. I do not know how grpc behaves if deadline is in the past.

File: OrbitClientServices/CrashManager.cpp (hunk 1 above)
Comment (IrinaShkviro) [reply OrbitClientServices/CrashManager.cpp]: Yes, you're right: I added a check for this, thanks!

File: OrbitClientServices/CrashManager.cpp (hunk 2 above)
Comment (IrinaShkviro) [reply OrbitClientServices/CrashManager.cpp]: Done, thanks. Probably if deadline in the past, it just do nothing: I set it to 1ms and OrbitService didn't crash (and no log records too).

File: OrbitClientServices/CrashManager.cpp [hunk 3]
```diff
@@ -40,9 +40,7 @@ void CrashManagerImpl::CrashOrbitService(
   grpc::Status status =
       crash_service_->CrashOrbitService(&context, request, &response);
 
-  if (!status.ok()) {
-    ERROR("Grpc call failed: %s", status.error_message());
-  }
+  CHECK(status.error_code() == grpc::StatusCode::DEADLINE_EXCEEDED);
```
Comment (dimitry-) [inline OrbitClientServices/CrashManager.cpp:43]: maybe have an if() with log - or even dialog if error_code is not DEADLINE_EXCEEDED. CHECK seems like an overkill here.

Comment (IrinaShkviro) [reply OrbitClientServices/CrashManager.cpp]: Added an error to the log

**Pull request merged**